src/types/organization.ts

```diff
@@ -49,18 +49,28 @@ export function fromPublicAdministrationToFoundAdministration(
     [pa.tipo_mail5, pa.mail5]
   ]
     .filter(([emailType, _]) => emailType === "pec")
-    .map(([_, pec]) => pec);
+    .reduce<{ [label: string]: string }>(
+      (prev, [_, pec], index) => ({
+        ...prev,
+        [String(index + 1)]: pec
+      }),
+      {}
+    );
   return FoundNotRegisteredAdministration.decode({
-    fiscalCode: pa.Cf,
-    ipaCode: pa.cod_amm,
-    legalRepresentative: {
-      familyName: pa.cogn_resp,
-      firstName: pa.nome_resp
+    fiscal_code: pa.Cf,
```

Comment:
lgtm ! In future I suggest to put this kind of refactors (ie. camelCase to snake_case) in a separate PR so we can merge the other ones faster